### PR TITLE
Update aka.ms links

### PR DESCRIPTION
**Pull request by michellemcdaniel**
Update the aka.ms links in dogfooding.md to match the new link structure.

### CODE REVIEW

File: docs/project/dogfooding.md [hunk 1]
```diff
@@ -185,181 +185,181 @@ $ bin\Debug\net6.0\win-x64\publish\App.exe
...
+[osx-arm64-version-6.0.X]: https://aka.ms/dotnet/6.0/daily/runtime-productVersion.txt
+[osx-arm64-installer-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-osx-arm64.pkg
+[osx-arm64-installer-checksum-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-osx-arm64.pkg.sha512
+[osx-arm64-targz-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-osx-arm64.tar.gz
+[osx-arm64-targz-checksum-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-osx-arm64.tar.gz.sha512
+[osx-arm64-nethost-targz-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-nethost-osx-arm64.tar.gz
+[osx-arm64-symbols-targz-6.0.X]: https://aka.ms/dotnet/6.0/daily/dotnet-runtime-symbols-osx-arm64.tar.gz
+
+[linux-x64-badge-6.0.X]: https://aka.ms/dotnet/6.0/daily/sharedfx_linux-x64_Release_version_badge.svg?no-cache
+[linux-x64-version-6.0.X]: https://aka.ms/dotnet/6.0/daily/runtime-productVersion.txt
```
Comment (danmoseley) [inline docs/project/dogfooding.md:234]: tried this link at random and it didn't work?
https://aka.ms/dotnet/6.0/daily/runtime-productVersion.txt

Comment (adiaaida) [reply docs/project/dogfooding.md:234]: Let me go check the promotion build. I saw that runtime had promoted, so just assumed these links would be good now.

Comment (adiaaida) [reply docs/project/dogfooding.md:234]: Oh. It looks like the workaround branch that runtime is using doesn't have the link flattening code in it yet. Will change this PR to not flatten, and then once runtime is back on arcade master for promotion, I can come back and update the links again.

**Comment by adiaaida** [comment]
@danmoseley The only links now that aren't working are the svgs. I can't tell how these are produced by the repo, no svgs were published in the publishing pipeline, and the latest version that was promoted to the old aka.ms link was a 6.0.0-alpha build. Did runtime stop producing the svgs?

File: docs/project/dogfooding.md [hunk 2]
```diff
@@ -185,181 +185,181 @@ $ bin\Debug\net6.0\win-x64\publish\App.exe
...
+[win-x64-symbols-zip-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/dotnet-runtime-symbols-win-x64.zip
+
+[win-x86-badge-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/sharedfx_win-x86_Release_version_badge.svg?no-cache
+[win-x86-version-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/runtime-productVersion.txt
+[win-x86-installer-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/dotnet-runtime-win-x86.exe
+[win-x86-installer-checksum-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/dotnet-runtime-win-x86.exe.sha512
+[win-x86-zip-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/dotnet-runtime-win-x86.zip
+[win-x86-zip-checksum-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/dotnet-runtime-win-x86.zip.sha512
+[win-x86-nethost-zip-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/dotnet-nethost-win-x86.zip
+[win-x86-symbols-zip-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtimedotnet-runtime-symbols-win-x86.zip
```
Comment (danmoseley) [inline docs/project/dogfooding.md:204]: this link does not work either.

Comment (danmoseley) [reply docs/project/dogfooding.md:204]: ah, 
```suggestion
[win-x86-symbols-zip-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/dotnet-runtime-symbols-win-x86.zip
```

**Comment by danmoseley** [comment]
Re SVG, I don't know what needs to be updated to make those new links work.

File: docs/project/dogfooding.md (hunk 2 above)
Comment (danmoseley) [reply docs/project/dogfooding.md]: hmm, that doesn't seem to work either.

You might consider just checking all your links with a quick script and curl.

**Comment by adiaaida** [comment]
I'll dig through the commit history. They stopped being published back in November.

**Comment by hoyosjs** [comment]
> Re SVG, I don't know what needs to be updated to make those new links work.

This is a known issue. It's tracked by https://github.com/dotnet/runtime/issues/48000#issuecomment-784584666

**Comment by adiaaida** [comment]
Thanks @hoyosjs!

File: docs/project/dogfooding.md (hunk 2 above)
Comment (adiaaida) [reply docs/project/dogfooding.md]: I wonder if that one even exists anymore. I don't see it being published.

Comment (adiaaida) [reply docs/project/dogfooding.md]: The last version of that file that was published was the 6.0.0-alpha.1.20562.6 version. Are you ok with me removing links in the table that are no longer being published?

**Comment by hoyosjs** [comment]
As for the symbols zip, if it was not intentional, it was fallout from the move to the shared SDK. That'd need investigation unless there's someone that knows off the top of their head. cc: @jkoritzinsky, @NikolaMilosavljevic

**Comment by adiaaida** [comment]
Both the symbols zip and nethost zips no longer work due to the move to the shared sdk.

**Review by ViktorHofer — APPROVED** [review]
Don't we need to also update the

**Review by ViktorHofer — COMMENTED** [review]
Disregard my previous approval, that wasn't intentional.

**Comment by ViktorHofer** [comment]
@hoyosjs can you please open a new issue for symbol zip so that we don't loose track of it?

File: docs/project/dogfooding.md (hunk 1 above)
Comment (ViktorHofer) [reply docs/project/dogfooding.md]: Let's just wait until the next arcade promotion is in?

Comment (adiaaida) [reply docs/project/dogfooding.md]: Sounds good.

**Comment by hoyosjs** [comment]
Tracking in https://github.com/dotnet/runtime/issues/49026

**Comment by hoyosjs** [comment]
@adiaaida The runtime. Quick ask nonetheless. Can you please remove the nethost, symbol, and badge links please? We'll need to add them again, but currently they point to stale assets and they do more harm than good. Thanks!

**Comment by adiaaida** [comment]
I will @hoyosjs. Will take care of that today.

**Comment by adiaaida** [comment]
I have updated the table generator to comment out the badge, symbols and nethost links, as I didn't want us to lose the knowledge of those or have to redo the table links. Also, I was able to update to the latest link structure since runtime's special publishing branch took an update. PTAL @hoyosjs @ViktorHofer.

**Review by ViktorHofer — APPROVED** [review]

**Comment by ViktorHofer** [comment]
A lot of failing legs (presumably temporarily because of an outage?). As the touched code isn't exercised in CI (besides the passing Markdownlint leg), merging.

**Pull request merged**